app/services/idv/steps/upload_step.rb

```diff
@@ -21,27 +21,14 @@ def call
         # app/views/idv/doc_auth/upload.html.erb
         if params[:type] == 'desktop'
           handle_desktop_selection
-        elsif params[:combined]
-          # The user was shown the new combined view and
-          # submitted a phone number to this step with feature flag on
-          # OR
-          # The user was originally shown the new combined view,
-          # but has submitted to a step with the feature flag off
-          # (50/50 from new to old)
-          handle_phone_submission
         else
-          handle_mobile_selection
+          return bypass_send_link_steps if mobile_device?
```

Comment:
Just want to flag that this line seems new. Is it handling something that wasn't being handled previously?